common/defs.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#ifndef __COMMON_DEFS_H__
+#define __COMMON_DEFS_H__
+
+#define TRYTE_ALPHABET "9ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+
+#define RADIX 3
+
+#define NUMBER_OF_TRITS_IN_A_TRYTE 3
+#define NUMBER_OF_TRITS_IN_A_BYTE 5
+
+#define TRYTE_VALUE_MAX 13
+#define TRYTE_VALUE_MIN -13
+#define TRYTE_WIDTH 3
+
+#define HASH_LENGTH_BYTE 48
+#define HASH_LENGTH_TRYTE 81
+#define HASH_LENGTH_TRIT 243
+
```

Comment:
can you leave comments to distinguish between sections?

Reply:
Sure